docs/reference/aggregation.md

```diff
@@ -532,15 +532,22 @@ By default, the top 10 terms with the most documents are returned. Larger values
 
 ###### **split_size**
```

Comment:
Yes `split_size` is not a good name. It's supposed to control the amount of data that's sent between nodes, so `node_size` probably? Plus an alias to `shard_size` since it's close enough.